src/bin.ts

```diff
@@ -203,18 +206,23 @@ const deploy = async ({ yes, bucket }: { yes: boolean; bucket: string }) => {
                 websiteConfig.WebsiteConfiguration.RoutingRules = routingRules;
             }
 
+            console.log('Updating bucket website', config.bucketName);
             await s3.putBucketWebsite(websiteConfig).promise();
+            console.log('Bucket website updated');
         }
 
         spinner.text = 'Listing objects...';
         spinner.color = 'green';
+        console.log('Listing objects');
```

Comment:
Doesn't this make the output kinda spammy with both ora and the console log reporting the same stuff?
Maybe time to ditch ora entirely, or replace it for something that logs nice output both on CI envs and 'normal' terms like gatsby CLI does as earlier discussed?

We can also make a separate issue about this and leave this for now if you feel this is out of scope.

Reply:
Ah, looks like I left in some debug code. I'll remove these as soon as I have a chance. We can fix `ora` later.